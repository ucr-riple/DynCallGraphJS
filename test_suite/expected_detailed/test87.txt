{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test87.js:13:10:13:61)': [
        'Person (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test87.js:1:1:8:2)'
    ],
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test87.js:14:1:14:10)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test87.js:10:25:12:2)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test87.js:11:10:11:26)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test87.js:2:20:4:4)'
    ]
}