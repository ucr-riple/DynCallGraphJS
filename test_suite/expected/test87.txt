{ '(test87.js:13:10:13:61)':
   [ '(test87.js:1:1:8:2)' ],
  '(test87.js:14:1:14:10)':
   [ '(test87.js:10:25:12:2)' ],
  '(test87.js:11:10:11:26)':
   [ '(test87.js:2:20:4:4)' ] }